### ISSUE TITLE
Duplicate tilde in example code listing

### ISSUE THREAD

**Issue by tttapa** [doxygen/doxygen#8711]
**Describe the bug**  
Example code that contains a tilde (bitwise NOT) has the tilde duplicated in the Doxygen HTML output.

**Expected behavior**  
The same number of tildes being in the HTML output as in the original source file.

**Screenshots**
```cpp
/**
 * @example example.cpp
 *
 * An example.
 */

int main() {
    int i = 0;
    int j = ~i;
    return j;
}
```
Generates:
![Screenshot from 2021-08-12 02-40-25](https://user-images.githubusercontent.com/21296729/129121308-4b3a069d-a333-4394-bdac-6106a2abff0c.png)

**To Reproduce**  
[**Doxygen-8711.zip**](https://github.com/doxygen/doxygen/files/6972113/Doxygen-8711.zip)

**Version**  
1.9.2 (a4958eb7f24b72062a07439235fa9345e81836a7), current master version, compiled from source just now. Ubuntu 20.04 x86_64.

**Comment by doxygen** [comment]
@tttapa Please verify if this commit fixes the problem for you.

**Comment by tttapa** [comment]
That does indeed fix it, thanks a lot for the quick response!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.2.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
